build.gradle

```diff
@@ -392,4 +392,10 @@ jmh {
     warmupIterations = 5
     iterations = 10
     fork = 2
-}
\ No newline at end of file
+}
```

Comment:
I think, this can be removed from your commit as this was made by upstream somehow. Could happen that his came in because of a newline fix at build.gradle?